modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentLauncher.java

```diff
@@ -172,10 +181,18 @@
 
         IOException ioCause = X.cause(e, IOException.class);
 
-        if (ioCause != null && "404".equals(ioCause.getMessage())) {
-            log.error("You are using outdated version of Web agent. Please download latest version of Web agent from Web console");
+        if (ioCause != null) {
+            if ("404".equals(ioCause.getMessage())) {
```

Comment:
Why this check not in `onError`

Reply:
We show different message for error and connection error event.